user/caribou/module/src/CLICpix2EventConverter.cc

```diff
@@ -149,8 +149,9 @@ bool CLICpix2Event2StdEventConverter::Converting(eudaq::EventSPC d1, eudaq::Stan
   }
   // drop everything after second t0 has been seen:
   if(t0_seen_>1) {
-      EUDAQ_WARN("Detected T0 " + std::to_string(t0_seen_) + " times.");
-      return false;
+      std::string message = "Detected T0 " + std::to_string(t0_seen_) + " times.";
+      EUDAQ_WARN(message);
```

Comment:
Maybe this EUDAQ_WARN is not even necessary...

Reply:
Yeah, remove it. Logging plus exception with message is a bit of an overkill I'd say.

Reply:
I agree, done.